src/pages/mentorship/index.tsx

```diff
@@ -1,13 +1,30 @@
 // path: /mentorship
+import { GetServerSideProps } from 'next';
 
-import { Typography } from '@mui/material';
+import { MentorshipPageProps } from '@utils/types';
+import { fetchData } from 'lib/api';
 
-const MentorshipPage = () => {
-  return (
-    <div>
-      <Typography variant="h4">Welcome to the Mentorship Page</Typography>
-    </div>
-  );
+const MentorshipPage = ({ mentorship }: MentorshipPageProps) => {
+  return <div>{JSON.stringify(mentorship)}</div>;
+};
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  try {
+    const combinedResponse = await fetchData('mentorship/overview');
```

Comment:
Can you create a type here? Would be a new type for the mentorship overview response and the footer (combined)